manifest.jenkins.yml

```diff
@@ -10,3 +10,49 @@ applications:
     - pz-postgres
     - pz-geoserver
     - pz-postgres-service-key
+    - pz-sso-geoaxis
+  env:
```

Comment:
Please move all "env" values to the application.properties file. 

There's a number of ways you can do this. Either create a new profile and add them to that application-profile.properties, or just add them to the default application.properties file. Up to you. But env variables in the manifest are hard to upkeep, so let's remove all of these env variables completely from this file.